### PR TITLE
Uxe 76 calendar tests

### PR DESCRIPTION
**Pull request by juliewongbandue**
- Added ChevronLeft icon
- Added a simple Calendar test
- Changed styled clickable svgs to buttons in calendar nav
- Updates click event on `<Day>` to resolve React warning: ```Expected `onClick` listener to be a function, instead got `false`.```

### CODE REVIEW

File: src/components/inputs/Dates/DateRange/DateRange.test.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      name: 'Start Date',
+    });
+    await userEvent.type(startDateInput, '09/27/2022');
+
+    const endDateInput = screen.getByRole('textbox', {
+      name: 'End Date',
+    });
+    await userEvent.type(endDateInput, '09/28/2022');
+
+    debug(endDateInput);
```
Comment (juliewongbandue) [inline src/components/inputs/Dates/DateRange/DateRange.test.tsx:57]: Both dates appear in this input value for some reason: '09/27/202209/28/2022'

File: src/components/inputs/Dates/Calendar/Calendar.test.tsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+  it('Renders with initial month', () => {
+    const date = new Date(2022, 8, 26);
+
+    renderWithThemeProvider(
+      <Calendar data-testid="calendar" initialMonth={date} />
+    );
+
+    const month = screen.getByRole('heading').innerHTML;
+    expect(month).toBe('September 2022');
```
Comment (kenneth-duverge) [inline src/components/inputs/Dates/Calendar/Calendar.test.tsx:40]: I think we should be able to use the `toHaveTextContent` matcher
```suggestion
    const month = screen.getByRole('heading');
    expect(month).toHaveTextContent('September 2022');
```

Comment (juliewongbandue) [reply src/components/inputs/Dates/Calendar/Calendar.test.tsx:40]: Ah! thanks!!

File: src/components/inputs/Dates/DateRange/DateRange.test.tsx (hunk 1 above)
Comment (juliewongbandue) [reply src/components/inputs/Dates/DateRange/DateRange.test.tsx:57]: Looks like they're separate elements... 🤔 : 
Screen Shot 2022-09-28 at 12 37 38 PM

**Review by cdebotton — APPROVED** [review]
Looks good!

**Pull request merged**